src/format/formatDiagnosticMessage.ts

```diff
@@ -15,10 +15,19 @@ const formatTypeOrModuleBlock = (_: string, prefix: string, code: string) =>
       : code
   );
 
-export const formatDiagnosticMessage = (message: string) =>
-  message
+const formatObjectLiterals = (message: string) => {
+  const objectLiteralRegex = /'(\{.*?\})'/g;
+  return message.replace(objectLiteralRegex, (_: string, p1: string) => {
+    return unstyledCodeBlock(p1);
+  });
+};
```

Comment:
This regex is too comprehensive for returning an unstyled code block, I think we should handle this case like all the other replaces.

The original issue looks like one of these regexes matches on the wrong closing `'`.
probably it's this replace because it's the most common:
![CleanShot 2023-04-19 at 02 37 49@2x](https://user-images.githubusercontent.com/5693018/232927547-1c001752-d4a7-4f31-b7dc-c295c003bfcd.png)
the `.` is optional so maybe we'll need to replace it by non optional `.` or whitespace